cidr2ip/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+## cidr2ip-1.0.0
```

Comment:
Is it safe for us to add an extra newline below this? I usually create separations like that.

Also, do we have to use two of the hash characters or can we go with 1 as well for these for detection via `changelog.sh`?

Reply:
Yes for both.
I'll update the `changelog.sh` accordingly.